.gitlab-ci.yml

```diff
@@ -536,6 +534,23 @@ release_circleci_runner:
   after_script:
     - docker rmi datadog/agent-buildimages-$IMAGE:v$CI_PIPELINE_ID-${CI_COMMIT_SHA:0:7} datadog/agent-buildimages-$IMAGE:latest
 
+.slack-notifier-base:
+  stage: notify
```

Comment:
Maybe it would be clearer to not indicate the stage here. Since this one is extended by only two jobs that have different stages (notify, availability_image). WDYT?

Reply:
I will use the template provider by `#ci-cd` then I will have to overwrite it only for the additional stage